messages/src/message.rs

```diff
@@ -130,6 +133,9 @@ pub enum HostRequest {
     /// transceivers' memory maps such as the temperature or power draw, for the
     /// purposes of health and safety monitoring.
     ManagementInterface(ManagementInterface),
+
+    /// Ask the SP to return the available MAC addresses for host system use.
+    MacAddrs(MacAddrKind),
```

Comment:
I'm not sure including the `MacAddrKind` here makes sense: the SP technically knows whether it's a Gimlet / Sidecar / PSC, but it only has one block of MAC addresses to give in every case.

Reply:
Yeah, I wasn't sure whether we needed this. Just to make sure I understand, this particular request, which is from the Scrimlet host to the Sidecar SP, always returns the Sidecar block of addresses, i.e, (base, count = 256, stride = 1). Is that accurate? I guess I wondered if any other software would use this same mechanism, but since it's the transceivers task handling the request, which only runs on the Sidecar SP, then seems like the answer is "no".

If that's all accurate, I'll remove this when I bring this up to date with `main`.

Reply:
Good point that this is only answered by the `transceivers` task!

Yes, asking the Sidecar SP will only return the Sidecar's addresses – or rather, it will return whatever is in the VPD EEPROM (after the SP itself claims the first two addresses).  We're trusting that the VPD is configured correctly for the board that it's installed on.

Reply:
@mkeeter I rebased on `main` and removed the kind bits. Let me know how the Hubris side goes, if we need to update this at all!